### ISSUE TITLE
How to extend styles like styled-componets？

### ISSUE THREAD

**Issue by xh4528** [styils/styils#228]
Styils is very convenient to use, but it seems that there is no function to extend styles in the documentation. How can I extend styles like styled-components？Looking forward to a reply.

https://styled-components.com/docs/basics#extending-styles

**Comment by xh4528** [comment]
in vue3 , i have tried to pass other style components to styled, and the page appears to be working properly. The console prompts for some warnings, but adding CSS will fail after switching themes.
(requires manual addition `!important`)

![image](https://github.com/styils/styils/assets/41058583/623f3bcb-d6ba-4836-85e5-029a835f993a)


```
export const Menu = styled("div", (theme) => ({
  display: "flex",
  gap: "20px",
  position: "sticky",
  top: "0",
  color: theme?.color,
  backgroundColor: theme?.backgroundColor,
  transition: "linear .3s",
}));

export const NMenu = styled(Menu, (theme) => ({
  top: "100px",
}));
```
![Jun-23-2023 00-20-36](https://github.com/styils/styils/assets/41058583/0464a227-4d15-4c96-b549-ce473f461e04)

**Comment by zoy-l** [comment]
Hi, I will investigate.

**Comment by zoy-l** [comment]
@xh4528 Already fixed in v1.1.6, if there are still problems, please feel free to reopen or create a new issue.🤞

**Comment by xh4528** [comment]
@zoy-l  After updating v1.1.6, this issue has been resolved. Thank you for your help 👍
